packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx

```diff
@@ -15,10 +14,10 @@ const WardPatientCard: React.FC<WardPatientCardProps> = (props) => {
         <WardPatientCardRow key={i} {...props} />
       ))}
       <button
-        className={styles.wardPatientCardButton}
-        onClick={() =>
-          launchWorkspace<WardPatientWorkspaceProps>('ward-patient-workspace', { patientUuid: props.patient.uuid })
```

Comment:
Passing the type inside the `launchWorkspace` helps us to correctly determine the data to pass into the function, which is expected on the other side by the workspace.
Please update the type accordingly and add this back.
Thanks!